_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,145 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Forseti Notifier can send a variety of notifications to alert you
+of Forseti events. These notifications can be configured to be sent
+to different channels, and in different formats.
```

Comment:
You can configure notifications to send to different channels and in different formats.

Reply:
Done